owapi/heroes.py

```diff
@@ -0,0 +1,279 @@
+"""
+This contains the static content for the heroes api endpoint.
+"""
+HEROES = {
```

Comment:
Why is this a separate dictionary? You could easily combine the two dictionaries into one dictionary, that could even be generated from the other one.